source/administration/changelog.md

```diff
@@ -6,6 +6,8 @@ Also see [changelog in progress](https://bit.ly/2nK3cVf) for the next release.
 
 ## Release v5.33 - [Feature Release](https://docs.mattermost.com/administration/release-definitions.html#feature-release)
 
+- **v5.33.3, release day TBD**
+  - Fixing an issue where Office 365 returns different ID attributes based on user type, causing an error for users with expired sessions when they try to sign in to Mattermost. [MM-34356](https://mattermost.atlassian.net/browse/MM-34356)
```

Comment:
```suggestion
  - Fixing an issue where, after migrating to OpenID, Office 365 returns different ID attributes based on user type, causing an error for users with expired sessions when they try to sign in to Mattermost. [MM-34356](https://mattermost.atlassian.net/browse/MM-34356)
```

Reply:
Either of you can wordsmith, but this is only an issue after they do the OpenID migration.